src/modals/CenteredModal.tsx

```diff
@@ -44,39 +45,50 @@ const CenteredModal: FC<CenteredModalProps> = ({
   header,
   narrow = false,
   children
-}) => (
-  <ModalContainer onClose={onClose} focusMode={focusMode} hasPadding>
-    <CenteredBox
-      initial={{ opacity: 0, y: 20 }}
-      animate={{ opacity: 1, y: 0 }}
-      exit={{ opacity: 0, y: 20 }}
-      transition={{ duration: 0.2, ease: 'easeOut' }}
-      narrow={narrow}
-    >
-      <ModalHeader contrast={!!header}>
-        <TitleRow>
-          <PanelTitle smaller useLayoutId={false}>
-            {title}
-            {subtitle && <ModalSubtitle>{subtitle}</ModalSubtitle>}
-          </PanelTitle>
-          <CloseButton squared transparent onClick={onClose}>
-            <X />
-          </CloseButton>
-        </TitleRow>
-        {header && <ModalHeaderContent>{header}</ModalHeaderContent>}
-      </ModalHeader>
-      <ModalContent>{children}</ModalContent>
-      {isLoading && (
-        <>
-          <ModalBackdrop light />
-          <ModalLoadingSpinner>
-            <Spinner />
-          </ModalLoadingSpinner>
-        </>
-      )}
-    </CenteredBox>
-  </ModalContainer>
-)
+}) => {
+  const { t } = useTranslation('App')
+  const divRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    divRef?.current?.focus()
+  }, [divRef])
```

Comment:
I've seen this 3 times in the code, how about a hook?
```ts
const divRef = useFocusOnDivOnLoad()
```
or sth like that

Reply:
Yeah good idea